### PR TITLE
Add ability to set mod_nss listen port

### PR DESCRIPTION
**Pull request by rexcze**
I added ability to mod_nss to be able to change listen port from default 8443 to for example 443, which is default https(SSL) port. It is quite useful for us.

### CODE REVIEW

File: manifests/mod/nss.pp [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class apache::mod::nss (
   $transfer_log = "${::apache::params::logroot}/access.log",
   $error_log    = "${::apache::params::logroot}/error.log",
-  $passwd_file  = undef
+  $passwd_file  = undef,
+  $nss_port     = 8443
```
Comment (igalic) [inline manifests/mod/nss.pp]: is there  a special reason to call this parameter nss_port, instead of just port?

Comment (igalic) [inline manifests/mod/nss.pp]: could you please add a trailing , ?

**Comment by igalic** [comment]
@rexcze thank you very much for this patch.
it will however need a bit of work before we can accept it: can you please add documentation for this new parameter to the README
i'd also like to see your commits [squashed](https://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits).
*****
*n.b.:* squashing commits is probably easier if you follow the [github flow](https://guides.github.com/introduction/flow/index.html) work-flow…

**Comment by rexcze** [comment]
@igalic I do not know where to put documentation about port in nss module, because there is no documentation about nss module at all.

**Comment by igalic** [comment]
yeah, see… i was gonna say, and spec tests, but looked that up first and noticed there's no spec tests :((

**Comment by rexcze** [comment]
So, what to do now?

**Comment by igalic** [comment]
@rexcze first off, i'd like these commits to be [squashed down](https://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits) down to two or one commit. (see also my https://github.com/puppet-community/puppet-jira/pull/99#issuecomment-152528108 on how to do that, or join #puppet-dev / #puppet-community for hand-holding ;)

second: your commits currently shows up as a different author than github knows. if you want special magick internet points, you can either add that email to your [github settings](https://github.com/settings/emails), or change it with `git commit --amend --author="Your Name <your@email.address>"`

if you don't care about such superficial things, i'll then simply press the merge button :)

and, finally: https://tickets.puppetlabs.com/browse/MODULES-2809 https://tickets.puppetlabs.com/browse/MODULES-2808

**Comment by rexcze** [comment]
Hi, I did some magic (at least to me) and I hope it is squashed and author changed. If that magic did not help I do not care about internet points. :-)

**Comment by igalic** [comment]
@rexcze it doesn't appear to have worked :(
5-commits

**Comment by rexcze** [comment]
Ok, from my point of view it is easier to close this merge request, fork puppetlabs-apache again, make one big commit and request again.

**Pull request closed without merging**